eventmesh-trace-plugin/eventmesh-trace-zipkin/src/main/java/org/apache/eventmesh/trace/zipkin/ZipkinTraceService.java

```diff
@@ -144,12 +144,7 @@ public String get(Map<String, Object> carrier, String key) {
 
     @Override
     public void inject(Context context, Map<String, Object> map) {
-        textMapPropagator.inject(context, map, new TextMapSetter<Map<String, Object>>() {
-            @Override
-            public void set(@Nullable Map<String, Object> carrier, String key, String value) {
-                map.put(key, value);
-            }
-        });
+        textMapPropagator.inject(context, map, (carrier, key, value) -> map.put(key, value));
```

Comment:
It seems that the variable `map` in lambda should be `carrier`.

Reply:
Thanks for your feedback